test.js

```diff
@@ -0,0 +1,10 @@
+const marked = require('./');
+
+const md = `
+<!-- multi
+line
+comment
+-->
+`;
+
+console.log(marked(md).replace(/\r/g, '\\r').replace(/\n/g, '\\n'));
```

Comment:
@UziTech What's the purpose of this test? It doesn't have any assertions?

Reply:
oops that file was just for manual testing. I didn't mean to commit it.